internal/pkg/api/handleEnroll.go

```diff
@@ -197,10 +201,22 @@ func (et *EnrollerT) _enroll(
 ) (*EnrollResponse, error) {
 	var agent model.Agent
 	var enrollmentID string
+	var replaceToken string
 
 	span, ctx := apm.StartSpan(ctx, "enroll", "process")
 	defer span.End()
 
+	now := time.Now()
+
+	if req.ReplaceToken != nil && *req.ReplaceToken != "" {
+		replaceTokenBytes, err := bcrypt.GenerateFromPassword([]byte(*req.ReplaceToken), bcrypt.DefaultCost)
+		if err != nil {
```

Comment:
should we log error, looking at implementation of GenerateFromPassword it can be miriad of things

Reply:
Added a log message.